aws-samples/amazon-chime-sdk
# Update meeting manager join method with new v3 params

**Issue #:**

**Description of changes:**
With V3, `MeetingManager` `join` method params have changed. `MeetingProvider` params have also been updated. Refer to this PR for more information on updates to the React SDK: [751](https://github.com/aws/amazon-chime-sdk-component-library-react/pull/751)
**Testing**

1. How did you test these changes?
By using the demo application. I was able to join the meeting.
2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
Yes.
Steps:
- Join the meeting (Tests `MeetingManager.join()`)
- Test Voice Focus (Tests `MeetingProviderWrapper` and `MeetingProvider`)

VF Testing Steps:
- Open two Chrome tabs (no Safari, no FF)
- Disable speaker in one tab
- Disable microphone and set built in device as speaker in another tab
- Turn on Amazon Voice Focus in the first tab
- Apply a new device (AirPods)
- The current device changes to AirPods
- Then make some background noice, you should not hear any noice from the second tab since Amazon Voice Focus is enabled in the first tab.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
